src/main/java/org/apache/commons/imaging/formats/jpeg/iptc/IptcParser.java

```diff
@@ -248,6 +258,35 @@ public PhotoshopApp13Data parsePhotoshopSegment(final byte[] bytes,
         return elements;
     }
 
+    private String getEncodingCharsetName(byte[] codedCharacterSet){
+        final Character WHITESPACE = ' ';
+        String codedCharacterSetString = new String(codedCharacterSet);
+        try {
+            if (Charset.isSupported(codedCharacterSetString)) {
+                return codedCharacterSetString;
+            }
+        }catch (IllegalCharsetNameException e){
+
```

Comment:
Might be worth to add at least one comment saying why these exceptions are not important, and that can be safely ignored.